nebula-logger/tests/log-management/classes/LogEntryEventHandler_Tests.cls

```diff
@@ -155,24 +156,23 @@ private class LogEntryEventHandler_Tests {
         System.assertEquals(logEntryEvent.TransactionId__c, log.TransactionId__c);
         System.assertEquals(logEntryEvent.UserLoggingLevel__c, log.UserLoggingLevel__c);
         System.assertEquals(logEntryEvent.UserLoggingLevelOrdinal__c, log.UserLoggingLevelOrdinal__c);
-        System.assertEquals(currentUser.Username, log.LoggedByUsername__c);
         // System.assertEquals(currentUser.UserRoleId, log.UserRoleId__c);
         // System.assertEquals(currentUser.UserRoleId == null ? null : currentUser.UserRole.Name, log.UserRoleName__c);
         System.assertEquals(logEntryEvent.UserType__c, log.UserType__c);
 
         // Org fields
-        System.assertEquals(Url.getOrgDomainUrl().toExternalForm(), log.OrganizationDomainUrl__c);
-        System.assertEquals(orgEnvironmentType, log.OrganizationEnvironmentType__c);
-        System.assertEquals(org.Id, log.OrganizationId__c);
-        System.assertEquals(org.InstanceName, log.OrganizationInstanceName__c);
-        System.assertEquals(org.Name, log.OrganizationName__c);
-        System.assertEquals(org.NamespacePrefix, log.OrganizationNamespacePrefix__c);
-        System.assertEquals(org.OrganizationType, log.OrganizationType__c);
+        //System.assertEquals(Url.getOrgDomainUrl().toExternalForm(), log.OrganizationDomainUrl__c);
```

Comment:
did you want to keep these commented out?

Reply:
Whoops, I've uncommented & finished implementing them